controllers/volumereplicationgroup_controller.go

```diff
@@ -253,11 +253,12 @@ func filterPVC(mgr manager.Manager, pvc *corev1.PersistentVolumeClaim, log logr.
 // +kubebuilder:rbac:groups=ramendr.openshift.io,resources=volumereplicationgroups/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=ramendr.openshift.io,resources=volumereplicationgroups/finalizers,verbs=update
 // +kubebuilder:rbac:groups=replication.storage.openshift.io,resources=volumereplications,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=replication.storage.openshift.io,resources=volumereplicationclasses,verbs=get;list;watch;update;patch
+// +kubebuilder:rbac:groups=replication.storage.openshift.io,resources=volumereplicationclasses,verbs=get;list;watch
 // +kubebuilder:rbac:groups=storage.k8s.io,resources=storageclasses,verbs=get;list;watch
 // +kubebuilder:rbac:groups=core,resources=persistentvolumeclaims,verbs=get;list;watch;update;patch
 // +kubebuilder:rbac:groups=core,resources=persistentvolumes,verbs=get;list;watch;update;patch;create
 // +kubebuilder:rbac:groups=core,resources=events,verbs=get;create;patch;update
+// +kubebuilder:rbac:groups="",namespace=ramen-system,resources=secrets,verbs=get;list;watch
```

Comment:
I recall hearing that a ramen user can specify a namespace name where ramen objects are installed.  I suppose that process would include substituting the `ramen-system` namespace reference in `config/dr-cluster/rbac/role.yaml` with a user specified namespace name.  Is my understanding correct?

Reply:
This helps autogenerate the namespace as "ramen-system" in the role.yaml file that can be kustomized when installing ramen into another system. BUT, this should just be "system" and not "ramen-system" as in other places (although this does not seem to matter to the generators), and have changed it to be so.